2-js-basics/2-functions-methods/translations/README.fr.md

```diff
@@ -0,0 +1,200 @@
+
+À titre d'exemple, considérons [setTimeout](https://developer.mozilla.org/docs/Web/API/WindowOrWorkerGlobalScope/setTimeout), qui démarre une minuterie et exécutera le code une fois terminé. Nous devons lui dire quel code nous voulons exécuter. Cela ressemble à un travail parfait pour une fonction !
+
+Si vous exécutez le code ci-dessous, après 3 secondes, vous verrez le message **3 secondes se sont écoulées**.
+
+```javascript
+fonction displayTerminé() {
+  console.log('3 secondes se sont écoulées');
+}
+// la valeur de la minuterie est en millisecondes
+setTimeout(displayDone, 3000);
+```
+
+### Fonctions anonymes
+
+Jetons un autre regard sur ce que nous avons construit. Nous créons une fonction avec un nom qui sera utilisé une fois. Au fur et à mesure que notre application se complexifie, on se voit créer beaucoup de fonctions qui ne seront appelées qu'une seule fois. Ce n'est pas idéal. Il s'avère que nous n'avons pas toujours besoin de fournir un nom !
+
+Lorsque nous passons une fonction en tant que paramètre, nous pouvons éviter d'en créer une à l'avance et en créer une à la place dans le cadre du paramètre. Nous utilisons le même mot-clé `function`, mais à la place, nous le construisons en tant que paramètre.
+
+Réécrivons le code ci-dessus pour utiliser une fonction anonyme :
+
+```javascript
+setTimeout(fonction() {
+  console.log('3 secondes se sont écoulées');
+}, 3000);
+```
+
+Si vous exécutez notre nouveau code, vous remarquerez que nous obtenons les mêmes résultats. Nous avons créé une fonction, mais nous n'avons pas eu besoin de lui donner de nom !
+
+### Fonctions de la grosse flèche
+
+Un raccourci courant dans de nombreux langages de programmation (y compris JavaScript) est la possibilité d'utiliser ce qu'on appelle une fonction **flèche** ou **flèche grasse**. Il utilise un indicateur spécial de `=>`, qui ressemble à une flèche - d'où le nom ! En utilisant `=>`, nous pouvons ignorer le mot-clé `function`.
+
+Réécrivons notre code une fois de plus pour utiliser une fonction grosse flèche :
+
+```javascript
+setTimeout(() => {
+  console.log('3 secondes se sont écoulées');
+}, 3000);
+```
+
+### Quand utiliser chaque stratégie
+
+Vous avez maintenant vu que nous avons trois façons de passer une fonction en tant que paramètre et vous vous demandez peut-être quand utiliser chacune. Si vous savez que vous utiliserez la fonction plus d'une fois, créez-la normalement. Si vous ne l'utilisez que pour un seul emplacement, il est généralement préférable d'utiliser une fonction anonyme. Que vous utilisiez ou non une grosse fonction de flèche ou la syntaxe plus traditionnelle de « fonction » dépend de vous, mais vous remarquerez que la plupart des développeurs modernes préfèrent « => ».
+
+---
+
+## 🚀 Défi
+
+Pouvez-vous articuler en une phrase la différence entre les fonctions et les méthodes ? Essaie!
+
+## Quiz post-conférence
+[Quizz post-conférence](https://happy-mud-02d95f10f.azurestaticapps.net/quiz/10)
```

Comment:
Have to add the localization parameter
[Quizz post-conférence](https://happy-mud-02d95f10f.azurestaticapps.net/quiz/10?loc=fr)

Reply:
I will review the file again and fix the bug ASAP sir.